pkg/virtctl/create/vm/vm.go

```diff
@@ -139,6 +150,93 @@ var flags = []string{
 	CloudInitNetworkDataFlag,
 	InferInstancetypeFlag,
 	InferPreferenceFlag,
+	VolumeImportFlag,
```

Comment:
This need to be added above `InferInstancetypeFlag` and `InferPreferenceFlag`. Can you move it to L149 (above `CloudInitUserDataFlag`) please?